soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -1340,6 +1340,33 @@ struct peci_it8xxx2_regs {
 #define CECOPSTS		ECREG(EC_REG_BASE_ADDR + 0x2E06)
 #define CECCRH			ECREG(EC_REG_BASE_ADDR + 0x2E07)
 
+/**
+ *
+ * (37xxh, 38xxh) USBPD Controller
+ *
+ */
+/* TODO: After pd driver is implemented, access these registers by reg property. */
```

Comment:
Why does the usbpd driver, or its lack of, prevent from implementing proper register accesses now? If it doesn't, then it should be done in this PR.

Reply:
Yes, currently lack of usbpd driver in zephyr code base, so I can't access register properly. I think if usbpd driver architecture is more clear in future, then we will re-visit here.